src/System.Private.Xml/src/System/Xml/Core/XmlConfiguration.cs

```diff
@@ -25,24 +25,8 @@ internal static class XmlConfigurationString
     }
 
     [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
-    internal sealed class XmlReaderSection : ConfigurationSection
+    internal sealed class XmlReaderSection
```

Comment:
Are these XMLConfiguration sections actually used still? Should we remove them completely from the code?